packages/selection/src/containers/FieldContainer.spec.js

```diff
@@ -48,15 +48,26 @@ describe('FieldContainer', () => {
   });
 
   describe('getInputProps', () => {
-    it('applies correct accessibility role', () => {
+    it('applies correct accessibility attributes', () => {
       const input = findInput(wrapper);
 
       expect(input).toHaveProp('id', `${CONTAINER_ID}--input`);
       expect(input).toHaveProp('aria-labelledby', `${CONTAINER_ID}--label`);
-      expect(input).toHaveProp(
-        'aria-describedby',
-        `${CONTAINER_ID}--hint ${CONTAINER_ID}--message`
+      expect(input).toHaveProp('aria-describedby', `${CONTAINER_ID}--hint`);
+    });
+
+    it('excludes aria-describedby if option is provided', () => {
+      wrapper = mount(
+        <FieldContainer id={CONTAINER_ID}>
+          {({ getInputProps }) => (
+            <div>
+              <input {...getInputProps({ 'data-test-id': 'input' }, { isDescribed: false })} />
+            </div>
+          )}
+        </FieldContainer>
       );
+
+      expect(findInput(wrapper)).toHaveProp('aria-describedby', null);
```

Comment:
Unfortunately no. The syntax above would expect the prop to have `undefined` as a value, but is null as provided.

If we were to provide `undefined` through the render-prop it would actually show as `undefined` in the DOM. With `null` React is removing it from the attribute list.